### ISSUE TITLE
Rust arg types for `loadXFromId` are incorrect

### ISSUE THREAD

**Issue by jedevc** [dagger/dagger#6933]
For example:

https://github.com/dagger/dagger/blob/3bc4bd1263570d67ced697d0b453c5d418ceb4ce/sdk/rust/crates/dagger-sdk/src/gen.rs#L5156

This type is wrong - it should instead be:

```diff
-     pub fn load_container_from_id(&self, id: Container) -> Container {
+     pub fn load_container_from_id(&self, id: ContainerID) -> Container {
```

The top-level `Query.Container` has [been deprecated for some time](https://github.com/dagger/dagger/blob/3bc4bd1263570d67ced697d0b453c5d418ceb4ce/docs/docs-graphql/schema.graphqls#L2111-L2117), and I'd like to remove it for v0.11.0 - but since `load_container_from_id` takes the wrong type, existing instances can't be replaced.

cc @dagger/sdk-rust

**Comment by kjuulh** [comment]
@jedevc this should now be fixed in #7038 We are still calling `.id()` on the struct I don't know if that is an issue, but getting out of that will require some more trickery. Let me know if that is the case.

**Comment by jedevc** [comment]
Should be fixed in #7038, closing! :tada:
